src/internal/finalize.cmd

```diff
@@ -0,0 +1,14 @@
+@setlocal
+@pushd %~dp0
+
+@set _C=Debug
+:parse_args
+@if /i "%1"=="release" set _C=Release
+@if not "%1"=="" shift & goto parse_args
+
+@echo Finalize build %_C%
+
+msbuild -Restore WixBuildFinalize\WixBuildFinalize.proj -p:Configuration=%_C% -nologo || exit /b
```

Comment:
Can we skip this for Debug?

Reply:
We could but it doesn't take long at all so it's good to ensure it doesn't break.

Reply:
CI always builds in Release. Running `build.cmd` in the root always runs Debug unless you take the effort to add Release.